packages/frontend/src/components/navigation/NavigationV2.js

```diff
@@ -0,0 +1,152 @@
+import styled from 'styled-components';
+
+import { WEP_DISABLE_ACCOUNT_CREATION } from '../../../../../features';
+import { WalletSelectorGetAWallet } from '../common/wallet_selector/WalletSelectorGetAWallet';
+import DesktopContainer from './DesktopContainer';
+import MobileContainer from './MobileContainer';
+
+const Container = styled.div`
+    &&& {
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        z-index: 1000;
+        border-bottom: 1px solid #F0F0F1;
+        background-color: #FFFFFF;
+        margin: auto;
+        @media (max-width: 991px) {
+            bottom: ${(props) => props.open ? '0' : 'unset'};
+            border: none;
+        }
+
+        h6 {
+            font-size: 13px;
+            margin-bottom: 5px;
+            color: #72727A;
+            font-weight: normal;
+        }
+
+        .account-selector {
+            padding: 0;
+            box-shadow: none;
+            border-radius: 0;
+        }
+    }
+`;
+
+const InnerContainer = styled.div`
+    &&& {
+        max-width: 1224px;
+        margin: 0 auto;
+    }
+`;
+
+export default ({
+    selectAccount,
+    showNavLinks,
+    flowLimitationMainMenu,
+    flowLimitationSubMenu,
+    refreshBalance,
+    availableAccounts,
+    account,
+    history,
+    onTransfer
+}) => {
+    const [menuOpen, setMenuOpen] = useState(false);
+    const [walletSelectorModal, setWalletSelectorModal] = useState();
+    const [showModal, setShowModal] = useState();
+
+    useEffect(() => {
+        if (menuOpen) {
+            document.addEventListener('keydown', handleKeyDown);
+            document.addEventListener('click', handleClick);
+        } else {
+            document.removeEventListener('keydown', handleKeyDown);
+            document.removeEventListener('click', handleClick);
+        }
+    }, [menuOpen]);
+
+    const handleKeyDown = useCallback((e) => {
+        if (e.keyCode === 27) {
+            setMenuOpen(false);
+        }
+    }, []);
+
+    const handleClick = useCallback((e) => {
+        const desktopMenu = document.getElementById('desktop-menu');
+        const mobileMenu = document.getElementById('mobile-menu');
+
+        if (e.target.tagName === 'SPAN') {
+            return false;
+        }
+
+        if (e.target.tagName === 'BUTTON' || e.target.tagName === 'A' || (!desktopMenu?.contains(e.target) && !mobileMenu?.contains(e.target))) {
+            setMenuOpen(false);
+        }
+    }, []);
+
+    const toggleMenu = useCallback(() => {
+        if (menuOpen) {
+            setMenuOpen(false);
+        } else {
+            setMenuOpen(true);
+        }
+    }, [menuOpen]);
+
+    const handleSelectAccount = useCallback((accountId) => {
+        selectAccount(accountId);
+        setMenuOpen(false);
+    }, []);
+
+    const handleOnClickCreateNewAccount = () => {
+        if (WEP_DISABLE_ACCOUNT_CREATION) {
+            setShowModal('more-near-wallets');
```

Comment:
The `/create` route is gone, should this be the only statement in `handleOnClickCreateNewAccount` i.e. show the modal with wallets?